src/cmd/create.go

```diff
@@ -646,6 +647,7 @@ func getServiceSocket(serviceName string, unitName string) (string, error) {
 }
 
 func pullImage(image, release string) (bool, error) {
+	logrus.Debugf("This is the image we're looking for %s", image)
```

Comment:
Nitpick: "Looking for image %s" will be better because it matches the logs elsewhere.

Reply:
Unresolving, because the log message is still the same.